domains/certificates/Query.API/API.IntegrationTests/Repositories/SlidingWindowStateTests.cs

```diff
@@ -28,9 +28,9 @@ public SlidingWindowStateTests(IntegrationTestFixture integrationTestFixture)
         dbContext.Database.EnsureCreated();
     }
 
-    private static MeteringPointSyncInfo CreateSyncInfo(string? gsrn = null) =>
+    private static MeteringPointSyncInfo CreateSyncInfo(string gsrn) =>
```

Comment:
I would take GSRN in as parameter, instead of string, so it's object from the start :)

Reply:
fixed